### PR TITLE
[RHACS] ROX15356 Documenting how to configure Azure token for ACS(CS)

### PR DESCRIPTION
**Pull request by agantony**
Version(s):
3.74, 4.0

Issue:
https://issues.redhat.com/browse/ROX-15356

Link to docs preview:
https://57625--docspreview.netlify.app/openshift-acs/latest/operating/manage-user-access/configuring-user-authentication-on-rhacs.html

SME review:
- [ ] SME has approved this change.

Additional information:
- Draft reviewed by Alex: https://docs.google.com/document/d/1RGp6oogpYJpLR2Q9TF3P59GvluIuxr5DfWzwJhddiF8/edit?usp=sharing
- Merge into `rhacs-docs` and cherry-pick into:
    - `rhacs-docs-4.0`
    - `rhacs-docs-3.74`

### CODE REVIEW

**Comment by agantony** [comment]
SME review needed: Merlin team

File: operating/manage-user-access/configuring-user-authentication-on-rhacs.adoc [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+To authorize users, {product-title-short} relies on mapping rules that examine the values of specific claims like, `groups`, `email`, `userid` and `name` from either the user ID token or UserInfo endpoint response in OpenID Connect (OIDC).
+
+If these details are absent, the mapping cannot succeed and the user does not get access to the required resources.
+Therefore, you need to ensure that the claims required to authorize users from your IDP, for example, `groups`, are included in the authentication response of your IDP to enable successful mapping.
+
+For example, to connect an Azure Active Directory (AD) to {product-title-short} using OIDC, see xref:../../operating/manage-user-access/configuring-user-authentication-on-rhacs.adoc#connecting-azure-ad-to-rhacs-using-oidc_user-authentication[Connecting Azure AD to RHACS using OIDC].
+
+[id="connecting-azure-ad-to-rhacs-using-oidc_{context}"]
+== Connecting Azure AD to {product-title-short} using OIDC
```
Comment (ivan-degtiarenko) [inline operating/manage-user-access/configuring-user-authentication-on-rhacs.adoc:24]: Here we say `using OIDC`, while below I see `for SAML applications` which are two different auth protocols.

Comment (agantony) [reply operating/manage-user-access/configuring-user-authentication-on-rhacs.adoc]: Updated title

File: operating/manage-user-access/configuring-user-authentication-on-rhacs.adoc [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+Ensure that {rh-rhacs-first} is connected to atleast one Identity Provider (IDP) for user authentication.
+
+You can set up your own IDP right after provisioning using one of the guides under `Operating -> Managing user access`.
+After the IDP authenticates you, {product-title-short} maps your account to roles that determine your permissions and access scope.
+
+To authorize users, {product-title-short} relies on mapping rules that examine the values of specific claims like, `groups`, `email`, `userid` and `name` from either the user ID token or UserInfo endpoint response in OpenID Connect (OIDC).
+
+If these details are absent, the mapping cannot succeed and the user does not get access to the required resources.
+Therefore, you need to ensure that the claims required to authorize users from your IDP, for example, `groups`, are included in the authentication response of your IDP to enable successful mapping.
```
[the remark this answers is not in the document]
Comment (agantony) [reply operating/manage-user-access/configuring-user-authentication-on-rhacs.adoc]: ok, in that case, we might have to place the Azure AD module, right after the "Claim mappings" section, WDYT? @ivan-degtiarenko @kcarmichael08

[the remark this answers is not in the document]
Comment (agantony) [reply operating/manage-user-access/configuring-user-authentication-on-rhacs.adoc]: Would you be cherry-picking your PR to 4.0? @ivan-degtiarenko

File: operating/manage-user-access/configuring-user-authentication-on-rhacs.adoc [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+[id="configuring-user-authentication-on-rhacs"]
+= Configuring user authentication on {product-title-short}
+include::modules/common-attributes.adoc[]
+:context: user-authentication
+
+toc::[]
+
+[role="_abstract"]
+
+Verify that {rh-rhacs-first} is connected to atleast one Identity Provider (IDP) for user authentication.
```
Comment (kcarmichael08) [inline operating/manage-user-access/configuring-user-authentication-on-rhacs.adoc:11]:
```suggestion
Verify that {rh-rhacs-first} is connected to at least one Identity Provider (IDP) for user authentication.
```

File: operating/manage-user-access/configuring-user-authentication-on-rhacs.adoc [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+After the IDP authenticates you, {product-title-short} maps your account to roles that determine your permissions and access scope.
+
+//-----------to be deleted--------------------
+////
+You can set up your own IDP right after provisioning using one of the guides under `Operating -> Managing user access`.
+////
+//----------to be deleted---------------------
+
+
+To authorize users, {product-title-short} relies on mapping rules that examine the values of specific claims like, `groups`, `email`, `userid` and `name` from either the user ID token or UserInfo endpoint response in OpenID Connect (OIDC).
```
Comment (kcarmichael08) [inline operating/manage-user-access/configuring-user-authentication-on-rhacs.adoc:23]:
```suggestion
To authorize users, {product-title-short} relies on mapping rules that examine the values of specific claims like `groups`, `email`, `userid` and `name` from either the user ID token or the `UserInfo` endpoint response in OpenID Connect (OIDC).
```

File: operating/manage-user-access/configuring-user-authentication-on-rhacs.adoc [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+//----------to be deleted----------------
+////
+For example, to connect an Azure Active Directory (AD) to {product-title-short} using OIDC, see xref:../../operating/manage-user-access/configuring-user-authentication-on-rhacs.adoc#connecting-azure-ad-to-rhacs-using-oidc_user-authentication[Connecting Azure AD to {product-title-short} using OIDC].
+////
+//-----------to be deleted--------------------
+
+[id="connecting-azure-ad-to-rhacs-using-sso-configuration_{context}"]
+== Connecting Azure AD to {product-title-short} using SSO configuration
+
+
```
Comment (kcarmichael08) [inline operating/manage-user-access/configuring-user-authentication-on-rhacs.adoc:40]: delete blank line

**Review by kcarmichael08 — COMMENTED** [review]
couple of minor comments; otherwise LGTM!

File: operating/manage-user-access/configuring-user-authentication-on-rhacs.adoc (hunk 3 above)
Comment (agantony) [reply operating/manage-user-access/configuring-user-authentication-on-rhacs.adoc:11]: Done

File: operating/manage-user-access/configuring-user-authentication-on-rhacs.adoc (hunk 5 above)
Comment (agantony) [reply operating/manage-user-access/configuring-user-authentication-on-rhacs.adoc:40]: Done

File: operating/manage-user-access/configuring-user-authentication-on-rhacs.adoc (hunk 4 above)
Comment (agantony) [reply operating/manage-user-access/configuring-user-authentication-on-rhacs.adoc:23]: Done

**Comment by agantony** [comment]
Azure AD content was added to docs as part of Ivan's PR: https://github.com/openshift/openshift-docs/pull/56937 
cc: @gaurav-nelson @kcarmichael08 
Closing this PR.

**Pull request closed without merging**